azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -175,7 +150,16 @@ public Props getProps() {
   }
 
   public List<Project> getProjects() {
-    return new ArrayList<Project>(projectsById.values());
+    List<Project> projects;
```

Comment:
This may not be required since you either populate the project or you throw a runtime exception.